include/rmm/mr/device/logging_resource_adaptor.hpp

```diff
@@ -108,11 +116,17 @@ class logging_resource_adaptor final : public device_memory_resource {
     init_logger(auto_flush);
   }
 
-  logging_resource_adaptor()                                               = delete;
-  ~logging_resource_adaptor() override                                     = default;
-  logging_resource_adaptor(logging_resource_adaptor const&)                = delete;
-  logging_resource_adaptor& operator=(logging_resource_adaptor const&)     = delete;
-  logging_resource_adaptor(logging_resource_adaptor&&) noexcept            = default;
+  logging_resource_adaptor()                                           = delete;
+  ~logging_resource_adaptor() override                                 = default;
+  logging_resource_adaptor(logging_resource_adaptor const&)            = delete;  ///< non-copyable
+  logging_resource_adaptor& operator=(logging_resource_adaptor const&) = delete;  ///< non-copyable
```

Comment:
Be consistent about how these are written. Some places use `non-copyable` and others use `Non-copyable`.
```suggestion
  logging_resource_adaptor(logging_resource_adaptor const&)            = delete;  ///< Non-copyable
  logging_resource_adaptor& operator=(logging_resource_adaptor const&) = delete;  ///< Non-copyable
```

Reply:
Why do we need these comments at all? It's obvious if you delete the copy ctor and copy assignment operator that objects of the class can't be copied.

Reply:
This is indeed just to avoid warnings. I couldn't find easy ways to suppress warnings on specific _types_ of symbols (e.g. deleted constructors), only on things like string matching of symbols. I can look again though. I agree that this is ugly, but I think the tradeoff for a warning-free docs build that we can use to identify real problems in our docs is worth it (if that's what it takes).

Reply:
Cons:
* More text to maintain
* Some of us use the docs from the code directly, not HTML docs, and this adds clutter
* Arguably adds clutter to the HTML docs too.

Reply:
It looks like I can remove the docs on deleted operators!